srt-tokio/tests/message_splitting.rs

```diff
@@ -12,33 +12,36 @@ const PACKET_SIZE: usize = 1 << 19;
 async fn message_splitting() -> Result<()> {
     let _ = pretty_env_logger::try_init();
 
-    let sender = SrtSocketBuilder::new_connect("127.0.0.1:11124").connect();
+    let sender = SrtSocketBuilder::new_connect("127.0.0.1:11124")
+        .latency(Duration::from_secs(2))
```

Comment:
Any particular reason for 2s latency here?

Reply:
Yeah good question--as messages can potentially be unbounded in size and shutdown should end eventually. I'm not sure that linking it to TSBPD is necessary though--maybe link with rtt? 

Another thing is senders should probably cancel the current message if it receives a shutdown. 

Willing to be convinced here

Reply:
I felt that waiting for the TSBD latency duration in order to shutdown allows for the receiver to drain any outstanding packets without losing data or real time playback. I'm fine with canceling current in-flight messages on shutdown, but messages that are waiting for playback should be drained I think. Which requires waiting for roughly TSBD latency